wire-ios/Wire-iOS/Sources/UserInterface/Location/LocationSelectionViewController.swift

```diff
@@ -73,81 +98,95 @@ final class LocationSelectionViewController: UIViewController {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        if !userLocationAuthorized { mapView.restoreLocation(animated: true) }
-        locationManager.requestWhenInUseAuthorization()
-        updateUserLocation()
-
+        if !userLocationAuthorized { mapViewController.mapView.restoreLocation(animated: true) }
```

Comment:
Please pass the provided `animated` argument.

```suggestion
        if !userLocationAuthorized {
            mapViewController.mapView.restoreLocation(animated: animated)
        }
```